app/assets/stylesheets/employees.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the employees controller here.
```

Comment:
Given the commit message, I'd expect to see here the table migration and the schema change